### DISCUSSION TITLE
Cannot register as a new SDK developer

### DISCUSSION THREAD

**Discussion by hmmhmmhm** [pixiv/vroid-sdk-developers#96] [Q&A]
I tried to leave an inquiry to register as a new SDK developer, but an error prevented me from leaving an inquiry.

![523423423](https://github.com/pixiv/vroid-sdk-developers/assets/38493722/b4c98062-c6bc-4893-838c-196a638e7d87)

Also, when I try to report an error, the error report is not being sent.

![15414](https://github.com/pixiv/vroid-sdk-developers/assets/38493722/fee66e42-74dc-4d07-813b-da9ac0fb328b)

We are an incorporated company and would like to make a business contact regarding the SDK.
Can you give us another way to contact you?

pm2@kakao.com

**Comment by nyamadan** [comment]
Were you unable to send your application through [this Inquiry form](https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk&subject=VRoid)?
I will let the person in charge know about this.

Could you wait several days and post again?
